plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-actions-block/block.tsx

```diff
@@ -30,14 +33,28 @@ const Block = ( {
 	placeOrderButtonLabel: string;
 } ): JSX.Element => {
 	const { paymentMethodButtonLabel } = useCheckoutSubmit();
-	const label = applyCheckoutFilter( {
+
+	const store = select( CART_STORE_KEY );
```

Comment:
This is a block, subscribe to that exact value using `useSelect` instead of using `select` to avoid needless rerenders, thou this using both seems to have the same amount of rerenders as to not using any, indicating a problem higher up. Still, it's good practice to use the hooks here.

Reply:
Thanks for pointing this out, Nadir. I've adjusted this part accordingly.